src/plugins/intel_gpu/src/graph/impls/ocl/custom_primitive.cpp

```diff
@@ -261,6 +295,7 @@ static std::unique_ptr<primitive_impl> create(const custom_gpu_primitive_node& a
 namespace detail {
 
 attach_custom_gpu_primitive_impl::attach_custom_gpu_primitive_impl() {
+    implementation_map<custom_gpu_primitive>::add(impl_types::ocl, shape_types::dynamic_shape, create, {});
```

Comment:
I'd suggest reverting this dynamic impl registration changes, as it's not sufficient to support shape-agnostic kernel logic. It will only make the pipeline think it's requesting dynamic kernel, while it will still get static one - which could lead to issues. So let's merge the initial version first, and then extend it with proper dynamic support

Reply:
You are right, when I added more dynamic dim to test case, I got some problems.
I just reverted dynamic shape kernel register. Take it as next step.